doc/userguide/capture-hardware/dpdk.rst

```diff
@@ -239,3 +239,71 @@ Encapsulation stripping
 Suricata supports stripping the hardware-offloaded encapsulation stripping on
 the supported NICs. Currently, VLAN encapsulation stripping is supported.
 VLAN encapsulation stripping can be enabled with `vlan-strip-offload`.
+
+Drop filter
+-----------------------------
+
+Drop filter can improve the performance of Suricata by filtering
+used-predefined flows directly in the Network interface card. The user can
+specify unwanted flows before the start of Suricata. These flows are not going to be
```

Comment:
Generally, I would avoid mentioning "flows" and focus more on the traffic patterns (or a more suitable term).
Potentially "traffic filter expression/rule"